libcobblersignatures/signatures.py

```diff
@@ -73,15 +72,19 @@ def signaturesjson(self) -> dict:
         """
         return self._signaturesjson
 
-    # TODO value has to be type dict or string
     @signaturesjson.setter
-    def signaturesjson(self, value: dict):
+    def signaturesjson(self, value: str):
+        """
+        The setter for the exported data structure.
+
+        :param value: The str which will be loaded by the Python json utility.
```

Comment:
Shouldn't we specify here that this could raise JSONDecodeError?

And then maybe add the note as `setter` at: https://github.com/cobbler/libcobblersignatures/pull/33/files#diff-eb577063602ce15a239faf678fbc96c0c21630a38ad26b823ed555b63319566cR70

Reply:
Fixed in 7f35e0c